### PR TITLE
Update Set-WinHomeLocation.md

### PR DESCRIPTION
**Pull request by mrbodean**
added space to create a clickable link to http://go.microsoft.com/fwlink/?LinkID=242308

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

File: docset/winserver2012-ps/internationalcmdlets/Set-WinHomeLocation.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The **Set-WinHomeLocation** cmdlet sets the value of the user GeoID object.
 The Windows GeoID setting is a user setting that describes the home location (that is, the country or region) of the current user account.
 Applications that require the current user account's home location, such as a driver for a television tuner application, can use this setting.
 
-A table of GeoIDs is available at Table of Geographical Locationshttp://go.microsoft.com/fwlink/?LinkID=242308.
+A table of GeoIDs is available at Table of Geographical Locations (http://go.microsoft.com/fwlink/?LinkID=242308).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/internationalcmdlets/Set-WinHomeLocation.md]:
```suggestion
A table of GeoIDs is available at [Table of Geographical Locations](http://go.microsoft.com/fwlink/?LinkID=242308).
```

**Comment by e0i** [comment]
@mrbodean Would you be able to follow-up with revision suggestions? We would like to proceed with merge. Thank you.

**Comment by e0i** [comment]
@mrbodean Closing as there has been no follow-up with the reviewers' revision suggestions. Please re-open the PR to proceed with the review. Thank you for your patience with the process and your contributions.

**Pull request closed without merging**